src/components/atoms/DatasetInfoBox.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Pane, Heading, Text, Badge } from 'evergreen-ui';
+
+type DatasetInfoBoxProps = {
+    title: string;
+    description: string;
+    tags: string[];
+};
+
+const DatasetInfoBox: React.FC<DatasetInfoBoxProps> = ({ title, description, tags }) => {
+    return (
+        <Pane width="100%" height="100%">
+            <Heading size={600}>{title}</Heading>
+            <Text overflow="auto">{description}</Text>
+            <Pane display="flex" flexWrap="wrap">
+                {tags.map((tag, index) => (
```

Comment:
The index should not be used as a key when mapping. In this case it makes no sense for the tags _not_ to be unique, so use them.

Reply:
good idea 💡